runtime/tests/checker/typeargument_test.go

```diff
@@ -178,6 +155,79 @@ func TestCheckTypeArguments(t *testing.T) {
 	})
 }
 
+type checkParameterizedTypeIsInstantiatedTestCase struct {
+	name string
+	code string
+}
```

Comment:
Move into `TestCheckParameterizedTypeIsInstantiated`, it does not have to be a global declaration.

Reply:
moved.